src/SourceBuild/content/test/Microsoft.DotNet.Tests/Directory.Build.props

```diff
@@ -0,0 +1,30 @@
+<Project>
+
+  <PropertyGroup>
+    <!-- Set manually as the project name doesn't follow the Arcade conventions. -->
+    <IsUnitTestProject>true</IsUnitTestProject>
```

Comment:
Not needed as the project file follows the convention

```suggestion
```

Reply:
You're right. Forgot to clean this after renaming the project.